### PR TITLE
cmd: install openssh if ssh-keygen is not available

### LINKED ISSUE #499: Building deploy key for EC2 provision fails
**Describe the bug**
On a Mac newly-updated to Mojave, building the deploy key after provisioning an EC2 instance crashed with the following error:
```
>> Step 2/4: Building deploy key...
2019/01/06 20:43:13 key generation failed: Process exited with status 127: bash: line 20: ssh-keyscan: command not found
```

**To Reproduce**
1. `inertia init`
2. `inertia provision ec2 $VPS`
3. Enter AWS credentials and pick an appropriate AMI.
4. Wait for step 1 to finish (provisioning and download of Docker/Inertia).
5. Crash occurs in step 2.

**Environment (please complete the following information):**
 - Client OS: macOS Mojave
 - VPS OS: EC2
 - Version: 0.5.0-rc1

### PR DESCRIPTION
**Pull request by bobheadxi**
:tickets: **Ticket(s)**: Closes #499

---

## :construction_worker: Changes

* lint keygen.sh
* if ssh-keygen isn't available, install openssh

## :flashlight: Testing Instructions

Use on a `Minimal` image in ec2 - see #499

### CODE REVIEW

File: client/scripts/keygen.sh [hunk 1]
```diff
@@ -7,17 +7,22 @@ set -e
 ID_DESTINATION=$HOME/.ssh/id_rsa_inertia_deploy
 PUB_ID_DESTINATION=$HOME/.ssh/id_rsa_inertia_deploy.pub
 
-# Check if destination file already exists:
-if [ -f $ID_DESTINATION ]; then
-    if [ ! -f $PUB_ID_DESTINATION ]; then
+# Install openssh if ssh-keygen is not available
+if hash ssh-keygen 2>/dev/null; then
```
Comment (rwblickhan) [inline client/scripts/keygen.sh]: What does `hash` do here?

**Comment by rwblickhan** [comment]
Looks good to me - I can try running it when I get home a bit later.

File: client/scripts/keygen.sh (hunk 1 above)
Comment (bobheadxi) [reply client/scripts/keygen.sh]: I'm not too sure, but @chadlagore used it [here](https://github.com/ubclaunchpad/inertia/blob/master/client/scripts/docker.sh#L41) to check for `curl`

Comment (rwblickhan) [reply client/scripts/keygen.sh]: Seems like it's a shell built-in the caches filenames in `$PATH`:
https://unix.stackexchange.com/questions/86012/what-is-the-purpose-of-the-hash-command

Comment (bobheadxi) [reply client/scripts/keygen.sh]: nifty!

**Pull request merged**